examples/helloWorld/README.md

```diff
@@ -112,6 +112,11 @@ sed -i 's/app: hello/app: my-hello/' \
     $BASE/kustomization.yaml
 ```
 
+*Note* `sed` works differently on different systems. On MacOS, for example, use this command instead:
+```
+sed -i '' 's/pineapple/kiwi/' $OVERLAYS/staging/map.yaml
+```
+
```

Comment:
Those aren't the right args at this point in the text.  How about just say:


> On a Mac, use:  
> ```
> sed -i '' $pattern $file
> ```
> to get in-place editing.

Reply:
Fixed. Sorry about fruit sneaking in there.